api/tests/unit/cat/answer_test.js

```diff
@@ -29,6 +29,17 @@ describe('Unit | Model | Answer', function() {
       // then
       expect(maxDifficulty).to.equal(5);
     });
+
+    it('should return level 2 if the challenge is undefined', function() {
```

Comment:
`Should return 2 for two challenge, when one of these is undefined`?

Reply:
C'est bien mon texte : si le challenge de l'answer est Undefined (vide, pas remonté par airtable), la maxDifficulty par default est 2 (j'ai retiré 'level'). C'était la maxDifficulty par défaut déjà présente)

Reply:
Ok mais c'est à deux parce qu'on lui en a donné deux nan ?

Reply:
C'est à deux car c'est la valeur par défaut des niveaux des skills